schemas/2017-04-01/Microsoft.ServiceBus.json

```diff
@@ -702,6 +702,40 @@
       ],
       "description": "Microsoft.ServiceBus/namespaces/AuthorizationRules"
     },
+    "namespaces_NetworkRuleSets_childResource": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.ServiceBus/Namespaces/NetworkRuleSets"
```

Comment:
>Microsoft.ServiceBus/Namespaces/NetworkRuleSets [](start = 13, length = 47)

for child resource, the type should be the short one - "NetworkRuleSets".

Reply:
resolved